packages/connect/src/data/DataManager.ts

```diff
@@ -49,8 +50,11 @@ export class DataManager {
         }
 
         // hotfix webusb + chrome:72, allow webextensions
-        if (this.settings.popup && this.settings.webusb && this.settings.env !== 'webextension') {
-            this.settings.webusb = false;
+        if (this.settings.popup && this.settings.env !== 'webextension') {
+            // allow all but WebUsbTransport
+            this.settings.transports = this.settings.transports?.filter(
+                (transport: string) => transport !== 'WebUsbTransport',
+            );
```

Comment:
So if I understand well here regardless of what `this.settings.webusb` says, webusb transport is not going to be included when using popup, unless it is a web extension ?

Reply:
yes, exactly. One of our goals is to find a way how to enable webusb for 3rd parties in web. It will be probably popup rendered within iframe